bwctest.sh

```diff
@@ -90,7 +89,9 @@ if [ -z "$OPENSEARCH" ]; then
     PLATFORM="${DASH_ARR[3]}"
     ARCH="${DOTS_ARR[0]}"
 
-    OPENSEARCH="https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/$VERSION/latest/$PLATFORM/$ARCH/dist/opensearch/opensearch-$VERSION-$PLATFORM-$ARCH.tar.gz"
+    OPENSEARCH="https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/$VERSION/latest/$PLATFORM/$ARCH/tar/dist/opensearch/opensearch-$VERSION-$PLATFORM-$ARCH.tar.gz"
 fi
 
-./scripts/bwctest_osd.sh -b $BIND_ADDRESS -p $BIND_PORT -s $SECURITY_ENABLED -c $CREDENTIAL -o $OPENSEARCH -d $DASHBOARDS -v $DEFAULT_VERSIONS
+# CI=1
```

Comment:
Do we need this line?

Reply:
Since we have had a 2.0 successful build in a while and the CI runner we have has changed a view versions I have logic that this uncommenting this line will ensure the expect browser. I will probably create a commit to uncomment this one on this PR. Because I expect it to work but it's hard to tell right now.

Reply:
Do we have an issue to track that?

Reply:
I actually move the logic up a little higher to allow us to check if this value is set or not so it allows us to run locally if we set this variable to 0